sw/device/lib/aes.h

```diff
@@ -0,0 +1,95 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _F_LIB_AES_H__
+#define _F_LIB_AES_H__
+
+#include <stddef.h>
+#include <stdint.h>
+
+/**
+ * Supported AES modes
+ */
+typedef enum aes_modes { AES_ENC = 0, AES_DEC = 1 } aes_modes_t;
```

Comment:
Enum names should be of the form `kConst`.

Reply:
Okay, I changed that. I thought we would allow ALL_CAPS for the enumerator names but obviously missed that we changed this rule.

Reply:
It's inherited from the Google C++ style guide; I think it's the one weird naming thing we actively inherit. (Allcaps is always a macro.)